chrome/content/zoteroocr.js

```diff
@@ -1,234 +0,0 @@
-// zoteroocr.js
```

Comment:
Please don't remove this file. It is required for a plugin which supports both Zotero 6 and Zotero 7.